crates/rome_js_analyze/src/analyzers/nursery/use_exponentiation_operator.rs

```diff
@@ -0,0 +1,303 @@
+                    .object()
+                    .ok()?
+                    .omit_parentheses()
+                    .as_reference_identifier()?
+                    .has_name("Math");
+                let has_pow = static_member_expr
+                    .member()
+                    .ok()?
+                    .as_js_name()?
+                    .value_token()
+                    .ok()?
+                    .token_text_trimmed()
+                    == "pow";
+
+                if has_math && has_pow {
+                    return Some(());
+                }
+            }
+            JsAnyExpression::JsComputedMemberExpression(computed_member_expr) => {
+                let has_math = computed_member_expr
+                    .object()
+                    .ok()?
+                    .omit_parentheses()
+                    .as_reference_identifier()?
+                    .has_name("Math");
+                let member_expr = computed_member_expr.member().ok()?;
+
+                let has_pow = match member_expr {
+                    JsAnyExpression::JsAnyLiteralExpression(literal_expr) => {
+                        literal_expr
+                            .as_js_string_literal_expression()?
+                            .inner_string_text()
+                            .ok()?
+                            .text()
+                            == "pow"
+                    }
+                    JsAnyExpression::JsTemplate(template) => {
+                        template.elements().len() == 1
+                            && template
+                                .elements()
+                                .into_iter()
+                                .next()?
+                                .as_js_template_chunk_element()?
+                                .syntax()
+                                .text_trimmed()
+                                == "pow"
+                    }
+                    _ => false,
+                };
+
+                if has_math && has_pow {
+                    return Some(());
+                }
+            }
+            _ => {}
+        }
+
+        None
+    }
+
+    fn diagnostic(ctx: &RuleContext<Self>, _: &Self::State) -> Option<RuleDiagnostic> {
+        let diagnostic = RuleDiagnostic::new(
+            rule_category!(),
+            ctx.query().range(),
+            "Use the '**' operator instead of 'Math.pow'.",
+        );
+
+        Some(diagnostic)
+    }
+
+    fn action(ctx: &RuleContext<Self>, _: &Self::State) -> Option<JsRuleAction> {
+        let mut mutation = ctx.root().begin();
+        let node = ctx.query();
+
+        if !should_suggest_fix(node)? {
+            return None;
+        }
+
+        let [base, exponent] = node.get_arguments_by_index([0, 1]);
+        let mut base_expr = base?.as_js_any_expression()?.clone().omit_parentheses();
+        let mut exponent_expr = exponent?.as_js_any_expression()?.clone().omit_parentheses();
```

Comment:
Is it possible to call `omit_parentheses` first. I believe that should remove the need to call clone

```suggestion
        let mut base_expr = base?.as_js_any_expression()?.omit_parentheses();
        let mut exponent_expr = exponent?.as_js_any_expression()?.omit_parentheses();
```

Reply:
We can't because `omit_parentheses` takes ownership of the ref :S